configure.ac

```diff
@@ -973,6 +973,22 @@ cat > conftest.c <<EOF
         darwin
 #elif defined(__VXWORKS__)
         vxworks
+#elif defined(__wasm32__)
+#  if defined(__EMSCRIPTEN__)
+	wasm32-emscripten
```

Comment:
Does it matter anywhere that these are the only "triplets" that are not actually a triplet but a pair? I know that `unknown` can be left out, but I just wanted to double-check there isn't some funky thing where this matters.

Reply:
AFAIK it does not matter. The check returns just ``darwin`` on Apple platforms or ``i386-gnu`` on GNU HURD.